src/main/rst/.templates/layout.html

```diff
@@ -152,19 +152,17 @@
 {%- block extrahead %}
 <script src="_static/code-snipplets.js" type="text/javascript"></script>
 <script src="_static/jquery.scrollTo-1.4.2-min.js" type="text/javascript"></script>
-<script src="_static/jquery.localscroll-1.2.7-min.js" type="text/javascript"></script>
```

Comment:
is it referenced anywhere else? if not... can you remove the file too?
https://github.com/SeleniumHQ/www.seleniumhq.org/blob/master/src/main/rst/.static/jquery.localscroll-1.2.7-min.js